### ISSUE TITLE
Auto-detect party code

### ISSUE THREAD

**Issue by samcal** [mkrum/Jamocracy#10]
If this is a user's second time using Jamocracy, they usually don't remember that they have to use the `!` command to remove themselves from their current playlist before adding another one. Our interface isn't very graceful about this, we need a better way, perhaps by auto-detecting if the user is trying to add themselves to a playlist.

**Comment by mkrum** [comment]
That is something that has been a consistent problem. We had talked about adding a timeout to the playlist, that would automatically remove the user after a few hours. I think this would work best, but would probably be the most work. If we wanted to just do auto detection, we would probably need to require that the input codes be submitted in all caps. This would be a much easier solution, but still imperfect

**Comment by samcal** [comment]
I disagree, I think that expected behavior is not auto-removal after a certain amount of time, especially after only a few hours. I think that if a user enters a party code, they expect to join that party, even if they are currently in a different party.

**Comment by mkrum** [comment]
I see what you're saying. What if we just added some symbol to the party joining process to distinguish from four letter songs. Such as:
!QCJS

**Comment by DanielKerrigan** [comment]
Also, when we generate a new party code, we aren't checking if that code is currently in use. Probability is  on our side, but we should probably fix it.

**Comment by samcal** [comment]
Yeah I like the idea of a new command, could even be longer than one symbol, like `join ABCD`

**Comment by mkrum** [comment]
I think we might want to look into adopting some symbol for jamocracy commands on the whole, just to make it easier to be able to identify what is a song and what is not.
